portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,6 +44,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String comment = getParameter(request, "comment-input", "");
```

Comment:
Do we want to add empty comments? Or should we just not add anything if it's not specified?